src/types/property.ts

```diff
@@ -0,0 +1,4 @@
+export type Property = {
+  name: string;
+  value: any;
```

Comment:
Our idea for the `properties` field on external events (and `metadata` on external sources) is that it's an area the user can input _any_ extra details that might be helpful to know about an event/source. That said - I want to say it would be okay to limit this field to a string as it's currently just being used for cosmetic/informative purposes. In the future, we might want to integrate the field into scheduling, so maybe expanding to something like `string | int` would be the way to go - definitely something good to discuss for us internally!

Reply:
Yeah that's understandable. It's not a hard requirement, but I'd rather have all the cases explicitly handled in the code just so that we know what to expect. `Property` seems somewhat similar to `FormParameter`, which also uses an `any` type for the `value`, but also comes with a `ValueSchema` that explicitly types it. Are you able to use `FormParameter` or portions of it instead?

Reply:
`FormParameter` definitely looks like what we want to use in the long-term! If we were to implement it, I think it would be best that we implement it in a very similar way to how `ActivityType` uses it - with the major difference being we would want to have a check on the database or ingest level to determine if all the parameters in `requiredParameters` are used (because External Events are immutable after ingest). With that said, is it OK to keep this `Property` type as-is for this PR, with the agreement that we'll update it in on our next PR/batch of work?

Reply:
If it's not possible to go with `FormParameter` right now, do you mind explicitly typing it to `string` | `number` | `object`? At least, that's what I'm understanding as the possible types based on how it's being used in `Properties.svelte`.